https://bugs.gentoo.org/949510#c4
https://github.com/wxWidgets/Phoenix/commit/de9aa4be5bb49adf82991c7582ea3c42ed505bf7.patch

From de9aa4be5bb49adf82991c7582ea3c42ed505bf7 Mon Sep 17 00:00:00 2001
From: Scott Talbert <swt@techie.net>
Date: Sun, 9 Feb 2025 17:50:08 -0500
Subject: [PATCH] Update SIP ABI version to 12.14

We haven't supported 12.8 since before we upgraded to SIP 6.6+.
However, SIP has not been correctly generating the correct module
version until very recently, so generating 12.8 doesn't actually compile
with current wxPython.
--- a/buildtools/config.py
+++ b/buildtools/config.py
@@ -88,7 +88,7 @@ class Configuration(object):
     PKGDIR = 'wx'
     # The name of the top-level package
 
-    SIP_ABI = '12.8'
+    SIP_ABI = '12.14'
     SIP_TRACE = False
 
     # ---------------------------------------------------------------
